src/Volume.jsx

```diff
@@ -32,7 +32,7 @@ export const Volume = ({ id, item, onChange, idx, removeitem, additem, options,
             </FormGroup>
             { options && options.selinuxAvailable &&
             <FormGroup className="pf-m-3-col-on-md" label={_("SELinux")} fieldId={id + "-selinux"}>
-                <FormSelect id={id + "-selinux"} className='pf-c-form-control'
+                <FormSelect id={id + "-selinux"} className='pf-v5-c-form-control'
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230601-052642-7ca6cf83-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)